src/lib/bootloader/sections.rb

```diff
@@ -22,10 +22,10 @@ def default
 
       return @default = "" if Yast::Stage.initial
 
-      saved = Yast::Execute.on_target("/usr/bin/grub2-editenv", "list", stdout: :capture)
+      saved = Yast::Execute.on_target("/usr/bin/grub2-editenv", "list", stdout: :capture) || ""
```

Comment:
Side note about `Yast::Execute.on_target`:
I have a nice little app to help me look up YaST API, http://mvidner-vm3.suse.cz:4567/?q=on_target
which helps me find http://www.rubydoc.info/github/yast/yast-yast2/Yast/Execute.on_target
but it does not say what the return value is and the link to Cheetah.run is not rendered. (we should quote the full URL, http://www.rubydoc.info/github/openSUSE/cheetah/Cheetah.run )
... and now I see that Cheetah does not mention the falsey return value either)